src/main.lua

```diff
@@ -205,6 +205,8 @@ function love.resize(w, h)
 	state.resize(w, h)
 end
 
+drawTimes = {}
+drawTimesCap = 300
```

Comment:
These are global variables for now. I want them to be part of Debug, but it hasn't yet been initialized. Currently it's initialized in inGame.